packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1417,6 +1417,21 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
   // Is this field in the current autofill context.
   bool _isInAutofillContext = false;
 
+  /// Whether to use a connection with the platform for text editing or not.
+  ///
+  /// Read-only input fields do not need a connection with the platform since
+  /// there's no need for text editing capabilities (e.g. virtualk keyboard,
+  /// etc).
+  ///
+  /// On the web, we always need a connection because we want some browser
+  /// functionalities to continue to work on read-only input fields like:
+  ///
+  /// - Relevant context menu.
+  /// - cmd/ctrl+c shortcut to copy.
+  /// - cmd/ctrl+a to select all.
+  /// - Changing the selection using a physical keyboard.
+  bool get _shouldUseConnection => kIsWeb || !widget.readOnly;
```

Comment:
Are we sure other engines also does not need such a feature? @justinmc @LongCatIsLooong 

If they need it, even if we don't use it now, we can change the wording.

Reply:
In other engines, we handle everything on the framework side for read-only fields (e.g. keyboard arrows/shortcuts, clipboard, context menu, etc). I tried it, it works fine. And since no one complained about it, I'm assuming read-only fields (and SelectableText) works fine on mobile platforms.

In any case, thanks for adding mobile people so we can confirm 😄

Reply:
I agree with @mdebbar.  I'll try to keep an eye out for use cases that might need this as we go forward though.